CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v1.229.0
+
+### Use semantic versioning scheme
+The agent will now utilize semantic versioning for subsequent releases. The previous version -- 1228 -- will carry forward into 1.229.0, and so on.
+
+### Use `web-vitals` library for internal timing calculations
+In an effort to align and standardize timings collecthe agent now uses the `web-vitals` library to track page timings such as CLS, LCP, FCP, TTFB and more. See https://www.npmjs.com/package/web-vitals for more information.
```

Comment:
`collecthe` - typo.

Also instead of "see https:// ....", you can just link to their repo like:
```[web-vitals](https://github.com/GoogleChrome/web-vitals)```